content/docs/guides/crosswalk/aws/vpc.md

```diff
@@ -484,11 +484,11 @@ raw resource so that you can code directly to the underlying AWS resource types,
 
 For information about configuring each of these resources, please refer to each type's API documentation:
 
-* [Vpc]({{< relref "/docs/reference/pkg/nodejs/pulumi/aws/ec2#Vpc" >}})
-* [Subnet]({{< relref "/docs/reference/pkg/nodejs/pulumi/aws/ec2#Subnet" >}})
-* [InternetGateway]({{< relref "/docs/reference/pkg/nodejs/pulumi/aws/ec2#InternetGateway" >}})
-* [NatGateway]({{< relref "/docs/reference/pkg/nodejs/pulumi/aws/ec2#NatGateway" >}})
-* [SecurityGroup]({{< relref "/docs/reference/pkg/nodejs/pulumi/aws/ec2#SecurityGroup" >}})
+* [Vpc]({{< relref "/docs/reference/pkg/aws/ec2/vpc" >}})
```

Comment:
The line directly above this seems to talk about "API" documentation. I think these links are fine, but I wonder if we should update that line to say just "documentation"? @justinvp thoughts?

Reply:
There are other pages that use the same wording, so should we update them to read "resource documentation" or just leave them as "API documentation"? It's just a little ambiguous to me because I personally refer to the API docs as the language specific  API docs and the resource docs are..well, the resource API docs, perhaps?

Reply:
I think it's fine to say `API documentation` as we're keeping all these under the `API Reference` section in the table of contents.